File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Reintegro/ReintegroDto.ts
// models/reintegro.dto.ts

export interface ReintegroDto {
    idReintegro: number;
    status: string;
    adminReintegroId: number | null;
    fechaReintegro: Date; // O Date, dependiendo de cómo manejes las fechas
    reintegroMotivo: string;
    nombreAdminReintegro:string;
    paymentId: number;
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/reintegro/ReintegroDto.java
package ps.com.viajeros.dtos.reintegro;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class ReintegroDto {
    private Long idReintegro;
    private String status;
    private Long adminReintegroId; // ID del administrador que gestionó el reintegro
    private String nombreAdminReintegro;
    private LocalDateTime fechaReintegro;
    private String reintegroMotivo;
    private Long paymentId; // ID del pago asociado
}
